### DISCUSSION TITLE
Security Releases feed or notifications

### DISCUSSION THREAD

**Discussion by jeroenj** [gravitational/teleport#34057] [Q&A] [answered]
From time to time a security patch for Teleport gets released upon we're presented in the Teleport app and CLI tools.

However, we'd like to be informed of such releases in the form of an RSS feed or mailing list as well. That way we can act as soon we're made aware of it.

I know we can track https://github.com/gravitational/teleport/releases but that's quite noisy given that various major versions are supported. Ideally we'd get a message when new security releases (for any major version) gets released.

**Comment by ravicious** [comment]
Unfortunately, at the moment there's no mailing list or an RSS feed that would let you receive notifications about security releases.
 
We are tracking this in https://github.com/gravitational/teleport/issues/34788.
 
FWIW, I see that our security fixes are generally "tagged" with `labels: security-patch=yes`, but those labels exist in the description of the releases only. You'd have to grab them from the GitHub API and use some tool to filter by them.

* https://api.github.com/repos/gravitational/teleport/releases
* https://api.github.com/repos/gravitational/teleport/releases/129644228 – this is an example security release, look at the end of the `body` field.

GitHub also has [a built-in Atom feed for releases](https://github.com/gravitational/teleport/releases.atom), but from what I see it doesn't include full descriptions of releases.

**Comment by jentfoo** [comment, marked as answer]
@jeroenj I am happy to report that we have updated our vulnerability disclosure processes.  We had a recent blog post detailing a lot of what we are changing: https://goteleport.com/blog/teleport-cve/

The short is that we have become a CNA and will be issuing CVE's for our vulnerabilities.  In addition we will be opening [GitHub Security Advisories on the Teleport Repo](https://github.com/gravitational/teleport/security/advisories).  This will allow you to watch the repo so that you can get proactive notifications whenever we publicly disclose a security release.

Let me know if you have any questions, thank you for raising this!
